.editorconfig

```diff
@@ -16,5 +13,19 @@ fsharp_max_array_or_list_width=120
 fsharp_max_infix_operator_expression=80
 fsharp_max_value_binding_width=120
 
+# Specific settings for "View/HTML" related files
+# It makes the code more consistent in term of spacing and indentation
 [GenerateHtml.fs]
-fsharp_experimental_elmish = true
\ No newline at end of file
+fsharp_experimental_elmish = true
+fsharp_array_or_list_multiline_formatter = number_of_items
+fsharp_max_array_or_list_number_of_items = 0
+fsharp_multi_line_lambda_closing_newline = true
+fsharp_multiline_bracket_style = aligned
+fsharp_keep_max_number_of_blank_lines = 1
+fsharp_align_function_signature_to_indentation = true
```

Comment:
I'm not seeing the effects from this in this file.

Reply:
`fsharp_multiline_bracket_style = aligned` is need to ensure the code inside of the DSL is aligned to the 4 spaces guides

![CleanShot 2025-01-02 at 11 09 38@2x](https://github.com/user-attachments/assets/17194b9d-29b9-44e7-885e-99a9310d9ffe)

`fsharp_keep_max_number_of_blank_lines = 1` has effet on code like that:

![CleanShot 2025-01-02 at 11 08 12@2x](https://github.com/user-attachments/assets/774162de-3e9b-4f5f-9731-7b9a8bc7c1f4)


`fsharp_align_function_signature_to_indentation = true` does nothing indeed, probably a left over copy/paste from experimentation/my own configuration. Will remove it